src/container/srv_target.c

```diff
@@ -86,6 +86,10 @@ cont_aggregate_epr(struct ds_cont_child *cont, daos_epoch_range_t *epr)
 	rc = vos_aggregate(cont->sc_hdl, epr, ds_csum_recalc, dss_ult_yield,
 			   (void *)cont->sc_agg_req);
 
+	/* Suppress csum error and continue on other epoch ranges */
+	if (rc == -DER_CSUM)
+		rc = 0;
```

Comment:
can add a ERROR log?

Reply:
There are already some D_ERROR() and D_DEBUG() messages for this -DER_CSUM error in the code path, so I didn't put more error messages here.
Actually, making vos_aggregate() returning -DER_CSUM is only for vos aggregate unit tests so far.